components/ide/code/codehelper/main.go

```diff
@@ -166,7 +166,6 @@ func getExtensions(repoRoot string) (extensions []Extension, err error) {
 		return
 	}
 	if config == nil || config.Vscode == nil {
-		err = errors.New("config.vscode field not exists")
```

Comment:
**Note (non-blocking):** For the _spring-petclinic_ repository I got this warning:
<img width="467" alt="image" src="https://user-images.githubusercontent.com/418083/184680901-cb44b312-4093-4392-8b52-60f92260a2bc.png">

Reply:
I think it is known bug for lang packs cc @jeanp413

Reply:
yeah different issue